mytwitter/settings.py

```diff
@@ -130,3 +130,12 @@
 
 # Authentication
 AUTH_USER_MODEL = 'user.CustomUser'
+LOGIN_URL = 'user:signin'
+
+# Email backend
+EMAIL_BACKEND = "django.core.mail.backends.filebased.EmailBackend"
+EMAIL_FILE_PATH = str(BASE_DIR.joinpath('sent_emails'))
```

Comment:
```suggestion
EMAIL_FILE_PATH = BASE_DIR / 'sent_emails'
```
もっと簡潔にかけます！
MEDIA_ROOTも同様